Debian and Ubuntu install a .gzipped version of the manual
Index: discrover-1.5.0/CMakeLists.txt
===================================================================
--- discrover-1.5.2.orig/CMakeLists.txt
+++ discrover-1.5.2/CMakeLists.txt
@@ -295,7 +295,7 @@ ENDIF()
 
 SET(MANUAL_LOCATION "not installed")
 IF(BUILD_MANUAL)
-  SET(MANUAL_LOCATION "${DOC_DIR}/discrover-manual.pdf")
+  SET(MANUAL_LOCATION "${DOC_DIR}/discrover-manual.pdf.gz")
 ENDIF()
 
 ADD_SUBDIRECTORY(src)
