--- src/CMakeLists.txt.orig	2015-01-19 12:53:38.350568803 +0100
+++ src/CMakeLists.txt	2015-01-19 12:52:19.761579290 +0100
@@ -1,5 +1,5 @@
 # add definitions, compiler switches, etc.
-ADD_DEFINITIONS(-Wall -funroll-loops -march=native -Wno-unused-local-typedefs)
+ADD_DEFINITIONS(-Wall -funroll-loops -Wno-unused-local-typedefs)
 
 # Allow to #include <discrover_config.hpp>, if doing out-of-source build.
 INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
